controller/network/command.go

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
Part of me wonders if there is a package in here for the command generics, command registry, etc. That could live on its own and be documented with an actual `package` level doc.

Reply:
Working on consolidating generic (not generics) things in the command package.